### ISSUE TITLE
ARIA support for Λrrow

### ISSUE THREAD

**Issue by nomisRev** [arrow-kt/arrow-website#145]
Got feedback on Mastodon that Λrrow is not user friendly in for screen readers, and should be wrapped in ARIA spans.

> You're using lamda in the name "Λrrow" to make it look cool, and I agree it looks cool, but a screen reader like VoiceOver will pronounce this as just "row". This could be fixed with <span aria-label="Arrow"> <span aria-hidden="true">Λrrow</span></span> at least, though I'm no ARIA expert.

I tried to do a quick sweep on the website, but was unsure how to deal with it in cases like yaml.

**Comment by raulraja** [comment]
Maybe this could be something we can change across the board and use `A`. The `Λ` can still be used in images or media when not presented in textual format.

**Comment by calvellido** [comment]
I'll have a look on the places where the content is being read from `.yml` files, I think it could a quick and very beneficial fix.

**Comment by calvellido** [comment]
Following on Raul, I have opened a PR in https://github.com/arrow-kt/arrow-website/pull/146 explaining what it could probably the best route to go.
